# **Exercise: Numbers in python**

1.You have a football field that is 92 meter long and 48.8 meter wide. Find out total area using python and print it.

In [1]:
length = 92
width = 48.8
area = length * width
print(area)

4489.599999999999


**Answer**: the area of football field is 4489.599999999999

**Explanation**:

We assign the length (92 meters) and width (48.8 meters) to variables.

We calculate the area using the rectangle formula:

Area
=
Length
×
Width
Area=Length×Width
Finally, we print the result, which is 4489.6 square meters.

2.You bought 9 packets of potato chips from a store. Each packet costs 1.49 dollar and you gave shopkeeper 20 dollar. Find out using python, how many dollars is the shopkeeper going to give you back?

In [3]:
totalPackets = 9
eachCost = 1.49
gaveShopkeeper = 20
return_cost = (gaveShopkeeper - (totalPackets * eachCost))
print(return_cost)

6.59


**Explanation:**
You bought 9 packets, each costing $1.49:

9×1.49=13.41

You gave the shopkeeper $20.

So, the change you should get back:

20−13.41=6.59

Answer: The shopkeeper will return $6.59.

3.You want to replace tiles in your bathroom which is exactly square and 5.5 feet is its length. If tiles cost 500 rs per square feet, how much will be the total cost to replace all tiles. Calculate and print the cost using python (Hint: Use power operator ** to find area of a square)

In [4]:
length = 5.5
cost_per_square_feet = 500
area = length ** 2
total_cost = area * cost_per_square_feet
print(total_cost)

15125.0


**Answer:** The total cost to replace all the tiles is ₹15125.0.

4.Print binary representation of number 17

In [5]:
print(bin(17))


0b10001


**Explanation:**

bin(17) converts the number 17 to binary.

The output starts with 0b, which indicates that it's a binary number.

So, 17 in binary is: 10001.